### PR TITLE
docs: add features that you concern

### PR DESCRIPTION
**Pull request by nicecui**

### CODE REVIEW

File: docs/faq-and-others/faq.md [hunk 1]
```diff
@@ -67,10 +63,6 @@ Because “grep” is the most useful command line tool on *nix platform to sear
 
 Yes, beginners can filter issues with ["good first issue"](https://github.com/GreptimeTeam/greptimedb/issues?q=label%3A%22good+first+issue%22) label. Additionally, more good first issues will be released on a rolling basis, so stay tuned!
 
-## Does GreptimeDB's storage system fully compatible with S3 object storage?
```
Comment (killme2008) [inline docs/faq-and-others/faq.md:70]: Added a link to docs/user-guide/concepts/architecture.md in FAQ too.

File: docs/faq-and-others/faq.md [hunk 2]
```diff
@@ -67,11 +63,10 @@ Because “grep” is the most useful command line tool on *nix platform to sear
...
-
-GreptimeDB's data access layer is based on [OpenDAL](https://github.com/apache/incubator-opendal), which supports most kinds of object storage services. Currently, GreptimeDB supports S3, OSS, and local disk storage. You can refer to storage configuration guide [here](https://docs.greptime.com/user-guide/operations/configuration#storage-option).
-
 ## Does GreptimeDB support dumping table-level data to S3?
 
 You can use the [`COPY TO` command](https://docs.greptime.com/reference/sql/copy#s3) to dump table-level data to S3.
 
+## TSDB features that you concern
+
+Please refer to [features that you concern](/user-guide/concepts/features-that-you-concern.md).
```
Comment (nicecui) [inline docs/faq-and-others/faq.md:72]: @killme2008 PTAL

**Review by killme2008 — APPROVED** [review]
LGTM

File: docs/user-guide/concepts/features-that-you-concern.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Features that You Concern
+
+## Does GreptimeDB support updates?
+
+Sort of. The updates can be effectively performed by insertions. If the primary keys have identical column values, the old data will be replaced with the new one. The primary keys in GreptimeDB are mostly like the series in other TSDBs.
```
Comment (xtang) [inline docs/user-guide/concepts/features-that-you-concern.md:5]:
```suggestion
Sort of. The updates can be effectively performed by insertions. If the primary keys have identical column values, the old data will be replaced with the new one. The primary keys in GreptimeDB are similar to the series in other TSDBs.
```

**Review by xtang — APPROVED** [review]

**Pull request merged**